external/src/main/java/com/redhat/cloud/policies/engine/lightweight/TriggersLoader.java

```diff
@@ -47,30 +53,58 @@ public List<FullTrigger> getTriggers(String accountId) {
     @CacheResult(cacheName = "account-latest-update")
     LocalDateTime findLatestUpdate(String accountId) {
         LOGGER.debugf("Finding latest triggers update for account %s", accountId);
-        String hql = "SELECT latest FROM AccountLatestUpdate WHERE accountId = :accountId";
-        try {
-            return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
-                    .setParameter("accountId", accountId)
-                    .getSingleResult();
-        } catch (NoResultException e) {
-            LOGGER.debugf("No latest triggers update found for account %s", accountId);
-            return LocalDateTime.MIN;
+
+        if (useOrgId) {
+            String hql = "SELECT latest FROM AccountLatestUpdate WHERE orgId = :accountId";
```

Comment:
Rename hql parameter to `:orgId`

Reply:
this is only a semantical thing, right? The left part is the property of the entity and the right on the parameter.

Reply:
Here is is cosmetics.

Reply:
That ui is still broken. I changed that code already, but it doesn't show there were changes. And the "viewed" checkbox bug is also still there, Gwenneg mentioned this mornin